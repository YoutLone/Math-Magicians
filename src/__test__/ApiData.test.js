import React from 'react';
import renderer from 'react-test-renderer';
import FetchQuote from '../components/ApiData';

describe('FetchQuote', () => {
  test('Match snapshot', () => {
    const component = renderer.create(<FetchQuote />);
    const instance = component.toJSON();
    expect(instance).toMatchSnapshot();
  });
});
