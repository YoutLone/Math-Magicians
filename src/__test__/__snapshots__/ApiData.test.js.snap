// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FetchQuote should match the snapShot 1`] = `
<div
  className="quotes"
/>
`;
